### PR TITLE
Results method for MCMC -- prints lovely summaries to the screen

### PR DESCRIPTION
**Pull request by ben18785**
Addresses #329.

### CODE REVIEW

**Comment by ben18785** [comment]
@MichaelClerx Any idea about the Travis error messages here? I've had a Google but it doesn't really make sense to me.

**Comment by ben18785** [comment]
@MichaelClerx Why? It's just an extension of diagnostic methods? Did you see the example notebook? It's not a results object - just a method that is called on the chains.  I don't like having it as an option for the controller object because, if you fail to select it, you then can't use the methods afterwards.

**Comment by ben18785** [comment]
So, to be clear, the MCMC still returns the same thing. All we are doing is wrapping the diagnostic methods into a useful helper function.

**Comment by MichaelClerx** [comment]
OK, sorry, I'll take a detailed look asap!

File: setup.py [hunk 1]
```diff
@@ -60,6 +60,7 @@ def load_version():
         # on systems without an attached display, it should never be imported
         # outside of plot() methods.
         'matplotlib>=1.5',
+        'tabulate>0.1',
```
Comment (MichaelClerx) [inline setup.py]: Why the version number?

File: pints/_mcmc/_results.py [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+#
+#
+from __future__ import absolute_import, division
+from __future__ import print_function, unicode_literals
+import pints
+import numpy as np
+from tabulate import tabulate
+
+
+class MCMCResults(object):
```
Comment (MichaelClerx) [inline pints/_mcmc/_results.py]: I'm wondering if there's a better name, like MCMCResultAnalysis, or MCMCResultSummary or something

Comment (MichaelClerx) [reply pints/_mcmc/_results.py]: MCMCSummary, MCMCDiagnosis, MCMCAnalysis ?

File: setup.py (hunk 1 above)
Comment (ben18785) [reply setup.py]: Removed version number

File: pints/_mcmc/_results.py (hunk 2 above)
Comment (ben18785) [reply pints/_mcmc/_results.py]: Yep, good idea! Have changed it to MCMCSummary.

**Comment by ben18785** [comment]
Great — cheers Michael!

**Pull request merged**